supportedBackends/sfs/src/main/scala/cromwell/backend/impl/sfs/config/ConfigHashingStrategy.scala

```diff
@@ -87,10 +91,61 @@ final case class HashPathModTimeStrategy(checkSiblingMd5: Boolean) extends Confi
   override val description = "hash file path and last modified time"
 }
 
-final case class HashFileStrategy(checkSiblingMd5: Boolean) extends ConfigHashingStrategy {
+final case class HashFileMd5Strategy(checkSiblingMd5: Boolean) extends ConfigHashingStrategy {
   override protected def hash(file: Path): Try[String] = {
     tryWithResource(() => file.newInputStream) { DigestUtils.md5Hex }
   }
 
-  override val description = "hash file content"
+  override val description = "hash file content with md5"
 }
+
+final case class HashFileXxH64Strategy(checkSiblingMd5: Boolean) extends ConfigHashingStrategy {
```

Comment:
I wonder whether it's possible to override the sibling md5 check for these strategies?

Reply:
Yes. All the sibling check does is read a string from a file with extension `.md5` it does not require any hashing at all. It is implemented in the ConfigHashingStrategy abstract class, so it also works for the new strategies.